docs/en-US/component/date-picker.md

```diff
@@ -181,9 +181,11 @@ Note, date time locale (month name, first day of the week ...) are also configur
 
 ## Methods
 
-| Method | Description               | Parameters      |
-| ------ | ------------------------- | --------------- |
-| focus  | focus the Input component | focusStartInput |
+| Method      | Description                  | Parameters |
+| ----------- | ---------------------------- | ---------- |
+| focus       | focus the Input component    | —          |
+| handleOpen  | opens the DatePicker popper  | —          |
+| handleClose | closes the DatePicker popper | —          |
```

Comment:
I think it's better to use `close` instead of `closes`, what do you think

Reply:
Completely agree 👌
I have just committed a fix for this.